src/js/themes/hpe.js

```diff
@@ -251,6 +251,20 @@ export const hpe = deepFreeze({
         },
         color: undefined,
       },
+      primary: {
+        // Designs call for using text-strong for the text color.
+        // Keeping the label color the same on hover to maintain readablility.
+        // color: hpe.global.colors['text-strong'].light,
```

Comment:
Think we still need to apply this label color, but we need to check how to get text-strong.light since we don't have access to `hpe` here I don't think.

Reply:
your right Ill fix